api/app/signals/apps/api/v1/validation/mixin.py

```diff
@@ -0,0 +1,19 @@
+from django.conf import settings
+
+from signals.apps.api.v1.validation.address.mixin import AddressValidationMixin
+
+
+class SignalValidationMixin(AddressValidationMixin):
+    @staticmethod
+    def feature_enabled(feature_flag_name=None):
+        return settings.FEATURE_FLAGS.get(feature_flag_name, True) if feature_flag_name else True
```

Comment:
We should document this behavior, a feature flag is assumed to be `True` when not set.